src/main/resources/db.changelog/changes/employee.yaml

```diff
@@ -0,0 +1,60 @@
+databaseChangeLog:
+  - logicalFilePath: no-path
+  - changeSet:
+      id: employee teble init
+      author: asilbekdev@gmail.com
+      preConditions:
+        onFail: MARK_RAN
+        not:
+          tableExists:
+            schemaName:
+            tableName: employee
+      changes:
+        - createTable:
+            tableName: employee
+            columns:
+              - column:
+                  name: em_id
+                  type: int
+                  autoIncrement: true
+                  constraints:
+                    primaryKey: true
+                    primaryKeyName: pk_em_id
+              - column:
+                  name: tg_info_id
+                  type: int
+                  constraints:
+                    unique: true
+                    nullable: true
+                    foreignKeyName: fk_tg_info
+                    references: tg_info(info_id)
+              - column:
+                  name: em_firstname
+                  type: varchar(255)
+              - column:
+                  name: em_lastname
+                  type: varchar(255)
+              - column:
+                  name: em_email
+                  type: varchar(255)
+                  constraints:
+                    unique: true
+              - column:
+                  name: em_position
+                  type: varchar(255)
+              - column:
+                  name: preferred_seat
```

Comment:
Maybe it should be fk?

Reply:
Yes It should be fk. But now I don't have Seat teble. that's why I leaved it that way. Is it okey or should I create one?

Reply:
Ok, than now it can be not fk